src/ui/components/Badge/styles.scss

```diff
@@ -12,6 +12,11 @@
     display: flex;
     flex-direction: row-reverse;
     margin: 0 0 12px;
+
+    .Addon-theme & {
+      flex-direction: row;
+      margin-left: 0;
```

Comment:
Normally this would need to be `@include margin-start(0);` but I think you can actually delete the margin style because it's a duplication of https://github.com/mozilla/addons-frontend/pull/5531/files#diff-434a89694ce53edc6a5cf18823ae07dcR27